css-flexbox/justify-content.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    #card-deck {
+      display: flex;
+    }
+
+    .card-container {
+      justify-content: center;
+      flex-basis: 25%;
+      padding: 10px;
+    }
```

Comment:
This doesn't look like it would work 🤔 

`justify-content` is a property applied to the flexbox container itself, not the children of a flexbox container. If you open this page in the browser, the output wouldn't match the example. Make sure to only turn in your work if the expected output is actually happening.